### DISCUSSION TITLE
Integration: Add Helix to other chain ecosystem bridge list

### DISCUSSION THREAD

**Discussion by 1022ranji** [ringdao-community-guild/discussion#134] [Integration]
- [ ]  https://portal.astar.network/astar/bridge [submitted, waiting reply]
- [ ]  https://syncswap.xyz/bridge
- [ ]  https://bridge.pancakeswap.finance/ 
- [ ]  https://scroll.io/ecosystem [In progress]
- [ ]  https://linea.build/apps?types=bridge [In progress]
- [ ]  https://app.optimism.io/retropgf-signup [In progress]
- [ ]  https://polygon.technology/ecosystem [In progress]
- [ ]  https://app.layer3.xyz/bridge [In progress]
- [x]  https://zksync.dappradar.com/ecosystem?page=1&category=defi_bridge [done]
- [x] https://portal.arbitrum.io/projects/bridges-and-on-ramps [done]

**Comment by 1022ranji** [comment]
@nblogist Could you please help with this integration?
About Helix: https://itering.notion.site/About-Helix-3328cfaac592499091cda1f74bdc74f6
<img width="671" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/11624842-a90f-4138-99cf-8aaf0a779536">

**Reply by nblogist** [reply]
On it @1022ranji !

**Comment by nblogist** [comment]
Hi @1022ranji adding the updates here! This will be a bit extended and multiple follow up task.
Please review some questions below and I can take it further!

1. < RANJI is waiting on response >
2. Messaged on SyncSwap Discord to find out the correct path, in order to list, since they didn't have anything standard. If no response then I will resurface the issue created by Ranji last year [here](https://github.com/syncswap/token-icons/issues/19) [Message](https://discord.com/channels/953301763811840000/953301766273925123/1239904969654538240)
3. @1022ranji how will the Helix and PancakeSwap combination look like? Can you pls share what is that you are envisioning. Thanks!
4. Scroll requires it being listed on 3 other platforms and then contacting their BD and then to fill a form. Form: https://tally.so/r/nGppyZ Launch Guide: https://scrollzkp.notion.site/Scroll-Launch-Guide-EXTERNAL-578355dda26f4b53ba2d4dc03513f3a4 - [This](https://zksync.dappradar.com/ecosystem?page=1&category=defi_bridge) on zk dapp Radar will help
5. < DONE ALREADY >
6. Linea doesn't have a proper channel as well, so submitted an issue [here](https://github.com/Consensys/linea-token-list/issues/155), this will need to be followed up - and different routes will be explored if this doesn't work. - Also @1022ranji Helix doesn't support Linea, so on what basis are we trying to request it being listed on ecosystem page?
7. @1022ranji I think you meant to list here: https://www.optimism.io/apps please lmk if i understood incorrectly. I have already requested and am waiting for response now!
![image](https://github.com/dcdao/discussion/assets/35498810/46ee06b6-c2e6-4c17-badb-34b5d81b0e58)
8. For Arbitirum I need this information to be added - basically a reference - @1022ranji pls help with that
![image](https://github.com/dcdao/discussion/assets/35498810/6f6c9df5-bc4e-4aa3-924b-9a1e4df92443) Then I can start the listing process [here](https://portal.arbitrum.io/projects/bridges-and-on-ramps)
9. Polygon is gilitching right now, will need to circle back on it next day.
10. @1022ranji can you share your vision on this as well? https://app.layer3.xyz/quests?tab=ecosystems shows the apps that have some sort of quests ongoing.

**Reply by 1022ranji** [reply]
Hi @nblogist,

3. Like [this](https://bridge.pancakeswap.finance/axelar) ⬇️ 
<img width="575" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/aaee8909-c8ca-405b-bff8-0381844e072d">

4. Helix has been added to Dapp Radar. I will next try to list Helix on two other platforms and then fill out the scroll form. Thanks for your guide.
<img width="759" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/ba325f42-5681-4744-8b37-3326ad5bd0d4">

6. Helix now supports Linea. Please refer to [this link](https://helixbridge.app/?token_category=eth&source_chain=linea&source_token=ETH&target_chain=arbitrum&target_token=ETH) for more details.
<img width="1197" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/bbfabfaa-8e24-48c4-8300-79dad33adb4e">

7. Yes, your understanding is correct. Please assist in listing Helix on https://www.optimism.io/apps.

10.There was a previous contact with Layer3, Helix would like to be listed [Here](https://app.layer3.xyz/bridge), there is a [form](https://airtable.com/appgk2DnSymuD296E/shr8T3uARmvUVeshZ) to fill out, but there is a requirement for a security audit, Helix is currently in the middle of a security audit, and I will be submitting this form after the audit is complete
<img width="547" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/f2501006-4d7e-4f17-9309-4bf9ecd5e588">

**Comment by nblogist** [comment]
1. Ranji do you have updates on this?
2. Sent several more messages since - if they still don't respond meaningfully then I think we should skip this or find a contact.
3. I messaged Pancake swap BD and they requested for a proposal - Created below and sent to their email info@pancakeswap.com .
```
**Proposal for Listing Helix Bridge on PancakeSwap**

Hi!

Hope you are doing well, I the Marketing and Branding Manager of Helixbridge.app upon request was directed to email here with a partnership proposal.
Introduction  
We are excited to propose the integration of Helix Bridge into PancakeSwap under the EVMs tab, Next to other bridges like Stargate and Wormhole. Helix Bridge is a cutting-edge cross-chain bridge that supports multiple assets and chains with unparalleled speed and low fees.

About Helix Bridge
Helix Bridge is designed to facilitate seamless cross-chain asset transfers with minimal costs. Key features include:  
- Multi-Asset Support: Supports USDT, Ethereum, Darwinia Ring Token, and Pink Token.  
- Wide Chain Compatibility: Compatible with Ethereum, Arbitrum, Polygon, Optimism, Linea, Scroll, Moonbeam, Darwinia, Astar zkEVM, and Blast.  
- Low Fees: Offers cross-chain swaps with fees as low as <$1.  
- Fast Transfers: Ensures rapid processing of cross-chain transactions.  
- Wallet Compatibility: Supports a variety of wallets including Rainbow, Coinbase Wallet, MetaMask, OKC, Rabby, and Talisman.

Benefits for PancakeSwap  
Integrating Helix Bridge into PancakeSwap will provide significant advantages:  
1. Enhanced User Experience: Users will benefit from fast, low-cost cross-chain swaps.  
2. Increased User Engagement: Offering a broader range of bridge options will attract more users to PancakeSwap.  
3. Competitive Edge: Position PancakeSwap as a leader in cross-chain interoperability by providing access to one of the industry's most advanced bridges.

Technical Overview 
Helix Bridge leverages advanced technologies to ensure security, efficiency, and ease of use:  
- Secure Architecture: Employs robust security protocols to protect user assets. [GitHub](https://github.com/helix-bridge)  
- Efficient Algorithms: Optimized for speed and cost-effectiveness.  
- User-Friendly Interface: Designed for simplicity, making it accessible to both novice and experienced users. [HelixBridge App](https://helixbridge.app)

Conclusion
By listing Helix Bridge on PancakeSwap, you will be providing your users with access to one of the most efficient and cost-effective cross-chain bridges available. This integration will not only enhance user satisfaction but also strengthen PancakeSwap's position as a leading decentralized exchange.

We look forward to the opportunity to discuss this proposal further and explore how we can work together to bring the benefits of Helix Bridge to the PancakeSwap community.

Contact Information  
For further details or to arrange a meeting, please respond back or contact:  
- Name: Furqan A.  
- Telegram: https://t.me/furqan

Thank you for considering our proposal. We are confident that this integration will be mutually beneficial and look forward to your positive response.

Sincerely,  
Furqan A.  
Marketing and Brand Manager  
Helix Bridge Team

```
4. Do you need any help on this?
5. I have now messaged Linea in discord to submit a project channel 
Please note that they required Audit links but I have told them that they are underway rn
![image](https://github.com/dcdao/discussion/assets/35498810/cdd932ef-ec5e-4b1b-bb8e-f84189d69051)
Submitted for Linea
![image](https://github.com/dcdao/discussion/assets/35498810/b0f1c651-563e-4b54-a8b7-07b77344faad)
7. As a follow up i have submitted to optimism again
![image](https://github.com/dcdao/discussion/assets/35498810/fe81cfe6-4c44-4682-8dda-25205e62732e)

Polygon is still not fixed:  I have messaged them on Discord to find out alternative ways [here](https://discord.com/channels/635865020172861441/897008314267889714/1259892572818510003)


please let me know if you have updates on any previous ones and if you need help on more! @1022ranji

**Reply by nblogist** [reply]
Responded to Polygon team, waiting on reply.

**Comment by nblogist** [comment]
Also completed a public Scroll expression of interest form: 
![image](https://github.com/dcdao/discussion/assets/35498810/e0072a8f-cc5b-4e8c-9802-ed5cbe0443bb)
